qa/tasks/vstart_runner.py

```diff
@@ -1398,8 +1403,9 @@ def get_logging_result_class():
 
 def launch_individually(overall_suite):
     no_of_tests_execed = 0
-
+    no_of_tests_failed, no_of_tests_execed = 0, 0
     LoggingResult = get_logging_result_class()
+    stream = LogStream(); stream.omit_result_lines = True
```

Comment:
It's generally frowned on to use semi-colons in Python, AFAIK.

Reply:
Fixed.